bundles/config/org.eclipse.smarthome.config.dispatch/src/main/java/org/eclipse/smarthome/config/dispatch/internal/ConfigDispatcher.java

```diff
@@ -38,39 +42,47 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * This class provides a mean to read any kind of configuration data from

```

Comment:
Sorry about this. I will revert to the original formatting, when I update the PR.